src/platform/Darwin/BlePeripheral.h

```diff
@@ -17,10 +17,29 @@
 
 #pragma once
 
+#if !__has_feature(objc_arc)
+#error This file must be compiled in ObjC++ mode with ARC.
+#endif
+
 #include <ble/Ble.h>
 
 #import <CoreBluetooth/CoreBluetooth.h>
 
-@interface MTRUUIDHelper : NSObject
-+ (CBUUID *)GetShortestServiceUUID:(const chip::Ble::ChipBleUUID *)svcId;
-@end
+namespace chip {
+namespace DeviceLayer {
+namespace Internal {
+
+// Holds a CBPeripheral and the associated CBCentralManager
+struct BlePeripheral
```

Comment:
This struct seems t o be completely unused?

Reply:
Ah yes that's a remnant of my attempt to use a shared_ptr as the BLE_CONN_OBJ type, will remove for now.

Reply:
Hm the whole header is actually unused, I think it slipped into a commit by accident.